marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/artifacts/mastering.sjs

```diff
@@ -0,0 +1,94 @@
+/**
+ Copyright (c) 2020 MarkLogic Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+'use strict';
+
+const DataHubSingleton = require('/data-hub/5/datahub-singleton.sjs');
+
+// define constants for caching expensive operations
+const dataHub = DataHubSingleton.instance();
+
+const collections = ['http://marklogic.com/data-hub/steps/mastering', 'http://marklogic.com/data-hub/steps'];
+const databases = [dataHub.config.STAGINGDATABASE, dataHub.config.FINALDATABASE];
+const permissions =
+  [
+    xdmp.permission(dataHub.consts.DATA_HUB_MATCHING_WRITE_ROLE, 'update'),
+    xdmp.permission(dataHub.consts.DATA_HUB_MATCHING_READ_ROLE, 'read')
+  ];
+const requiredProperties = ['name'];
+
+function getNameProperty() {
+    return 'name';
+}
+
+function getVersionProperty() {
```

Comment:
Can remove this and "requiredProperties" above too; can remove this function from matching and merging too

Reply:
Done. I had forgotten to remove the function from the exports so I was getting an error in bootstrap.